apps/fiona/fiona.cpp

```diff
@@ -1254,7 +1254,7 @@ getFoundCorrections(
     unsigned numCorrections = 0;
 
     TValue currentPos = firstCorrectionForRead[erroneousReadId];
-    while (currentPos != maxValue<TValue>())
+    while (currentPos != MaxValue<TValue>::VALUE)
```

Comment:
wow, I didn't know it was used like that. Are you sure this behaviour is intended? Maybe they meant `unknownValue<T>()` instead?